packages/template-retail-react-app/app/components/display-price/index.test.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {screen, within} from '@testing-library/react'
+import DisplayPrice from '../display-price/index'
+import {renderWithProviders} from '../../utils/test-utils'
+import mockProductDetail from '../../commerce-api/mocks/variant-750518699578M'
+
+const mockedBasePrice = /\$299\.99/i
+const mockedDiscountPrice = /\$149\.99/i
+
+describe('DisplayPrice', function () {
+    test('should render without error', () => {
+        renderWithProviders(<DisplayPrice product={mockProductDetail} scope="pdp" />)
+        expect(screen.getByText(mockedBasePrice)).toBeInTheDocument()
+        expect(screen.getByText(mockedDiscountPrice)).toBeInTheDocument()
+    })
+
+    test('should render according html tag for prices', () => {
+        const {getByTestId} = renderWithProviders(
+            <DisplayPrice product={mockProductDetail} scope="pdp" />
+        )
+        const basePriceTag = getByTestId('base-price')
+        const discountPriceTag = getByTestId('discount-price')
+        expect(within(basePriceTag).getByText(mockedBasePrice)).toBeDefined()
+        expect(within(discountPriceTag).getByText(mockedDiscountPrice)).toBeDefined()
+    })
```

Comment:
The first test is "base price is in the document (and same for discount price)".
The second test is "base price is in the (arbitrarily defined) `base-price` element in the document (and same for discount price)".

They seem to pretty much the same thing, what is the value of having both?

Reply:
I was just copying the tests that @alexvuong had done [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/8ae953d906e400541e29fbd6d1978c1d476650e5/packages/template-retail-react-app/app/components/display-price/index.test.js#L23-L26). i can remove, if needed. to me, the second test checks the styling of the two prices, while the first just checks for their existence.